Mathlib/GroupTheory/PushoutI.lean

```diff
@@ -267,7 +267,7 @@ instance : Inhabited (NormalWord d) := ⟨NormalWord.empty⟩
 
 instance (i : ι) : Inhabited (Pair d i) :=
   ⟨{ (empty : NormalWord d) with
-      head := 1,
+      head := 1, tail := _,
```

Comment:
```suggestion
      head := 1,
      tail := _,
```

Reply:
To be fully explicit, we could also replace this with
```
instance (i : ι) : Inhabited (Pair d i) :=
  ⟨{ head := 1,
     tail := empty.toWord,
     fstIdx_ne := fun h => by cases h
     normalized := empty.normalized }⟩
```